packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -68,22 +72,29 @@ export class MonacoEditorProvider {
             commandService.setDelegate(standaloneCommandService);
 
             return editor;
-        })));
+        });
+    }
+
+    protected getModelOptions() {
+        return {
+            tabSize: this.editorPreferences["editor.tabSize"]
+        };
     }
 
     protected async getEditorOptions(model: MonacoEditorModel): Promise<MonacoEditor.IOptions | undefined> {
```

Comment:
i don't think it should be async and it should return `MonacoEditor.IOptions | undefined`